spec/models/season/phase_switcher_spec.rb

```diff
@@ -4,10 +4,32 @@
 context 'when switching phases' do
   subject { Season.current }
 
+  describe 'destined' do
+    it 'sends the params to a phase setting method' do
+      phase = :fake_application_phase
+      Season::PhaseSwitcher.destined(phase)
+      expect(subject).to_not be_nil
+    end
+
+    it 'addresses the requested phase' do
+      wrong_phase = :fake_proposals_phase
+      Season::PhaseSwitcher.destined(wrong_phase)
+      expect(subject).to_not be_application_period
+    end
+
+    it 'fails silently when it receives a non-whitelisted phase' do
+      phase = :bad_intentions
+      bad_call = Season::PhaseSwitcher.destined(phase)
+      expect( bad_call ).to be_nil
```

Comment:
This doesn't test if `.destined` doesn't change things, it merely checks for the _return value_ of that method call. A better way is to make use of `expect { … }.not_to change { … }`. A good thing to check for in the `change` block could be `Season.current.updated_at`.